README.markdown

```diff
@@ -7231,6 +7233,40 @@ This feature was first introduced in the `v0.5.0rc1` release.
 
 [Back to TOC](#nginx-api-for-lua)
 
+tcpsock:receiveany
+------------------
+**syntax:** *data, err = tcpsock:receiveany(max?)*
```

Comment:
I think we should make the `max` argument required for the sake of security. Otherwise large volumn of incoming data would crash the LuaJIT VM.